### ISSUE TITLE
Planning and implementing a feature to change the app's language

### ISSUE THREAD

**Issue by andrewtavis** [scribe-org/Scribe-iOS#120]
### Terms

- [X] I have searched [open and closed feature requests](https://github.com/scribe-org/Scribe-iOS/issues?q=is%3Aissue+label%3Afeature)
- [X] I agree to follow Scribe-iOS' [Code of Conduct](https://github.com/scribe-org/Scribe-iOS/blob/main/.github/CODE_OF_CONDUCT.md)

### Description

This issue is to make the Scribe app react to the system language that's present on a user's device, thus allowing for localization of the app screen. Once this is finished, work will begin on creating localizations for all languages that Scribe currently provides keyboards for.

### Contribution

I'd be happy to work on this if others don't have interest :)

**Comment by gitpushOmnik** [comment]
Hey @andrewtavis, I would love to solve this issue. Can you please assign it to me?

**Comment by andrewtavis** [comment]
Yes, @gitpushOmnik :) Would be a great issue to work on 😊 Assigning you now!

**Comment by andrewtavis** [comment]
Hey @gitpushOmnik, answering your question [from Matrix](https://matrix.to/#/!NBSsugBSPLMaWNIuSq:matrix.org/$qINNeAQiMMmHJ92OzXAfI86nlvXmpAHY0VXHR8h-Zhg?via=matrix.org) here :)

> For issue #120, by changing the language of the app screen, do you mean changing the language of the privacy policy, keyboard installation and community texts according to the user's system set language? For example, if the system language was English, the text language of "privacy policy, keyboard installation and community" should be English and if the system language was German, the text language of "privacy policy, keyboard installation and community" should be German?

This is exactly what we're looking for in this issue. Within [Scribe/AppTexts](https://github.com/scribe-org/Scribe-iOS/tree/main/Scribe/AppTexts) we'd then be able to make new folders like `Spanish` and `German` where we'd have versions of the privacy policy and the app texts in each of those languages. The texts for the button and headers would also change as you discussed :) We'd have the `Privacy policy` button be localized to the given language, and with that also the keyboard installation steps and the text within the community section 😊

The ideality here would be that the English app texts would be the default, which we should be able to do by just assigning blank strings for the languages we want to allow. For languages that aren't English we check first to see if the string is empty, and if it is we assign the English version, or if not we assign the translation that we've gotten from the community :)

**Comment by andrewtavis** [comment]
Ok, @gitpushOmnik 😊 I think we're good to close this :) As I said in #266, let's hold off on adding new languages for now as the app will be going through an overhaul during GSoC. Nice to have two languages in there so that we have a basis to work from when we're adding more once the new designs are implemented.

Thanks so much for your help! Happy to have these contributions :)
